src/src/PluginStructs/P036_data_struct.cpp

```diff
@@ -540,20 +641,20 @@ uint8_t P036_data_struct::display_scroll(ePageScrollSpeed lscrollspeed, int lTas
 
     if (bLineScrollEnabled) {
       // settings for following line scrolling
-      if (PixLengthLineOut > getDisplaySizeSettings(_disp_resolution).Width) {
+      if (PixLengthLineOut > getDisplaySizeSettings(disp_resolution).Width) {
         ScrollingLines.Line[j].LastWidth = PixLengthLineOut; // while page scrolling this line is right aligned
       }
 
-      if ((PixLengthLineIn > getDisplaySizeSettings(_disp_resolution).Width) && (fScrollTime > 0.0f))
+      if ((PixLengthLineIn > getDisplaySizeSettings(disp_resolution).Width) && (fScrollTime > 0.0f))
```

Comment:
Comparing floats like this is dangerous.
Also where `fScrollTime` is used is in a divide, so if it is like 0.0000001, then the value of `dPix` may be just any value as it is only an `unit8_t`
So even a value of 0 is possible then.

Reply:
Done, see my next commit.
fScrollTime (float) changed to iScrollTime (int),  iScrollTime =scrollTime in ms